### DISCUSSION TITLE
Generators > .json files have a BOM; reason?

### DISCUSSION THREAD

**Discussion by Gounlaf** [dlemstra/Magick.NET#1535] [General] [answered]
:wave: 

I've notice all .json files in Native have a BOM. I'm just wondering why?

Thanks.

Regards.

**Comment by dlemstra** [comment, marked as answer]
I probably started with a file that contains a BOM :) There is no need for that. Please be aware that those files will be removed in the future because I will be going to replace it with source generators.

**Reply by Gounlaf** [reply]
> Please be aware that those files will be removed in the future because I will be going to replace it with source generators.

Good to know. I was planning to use them 😅
